test/PushNotificationTests/PushNotificationTests.testdef

```diff
@@ -0,0 +1,18 @@
+{
+    "Tests": [
+        {
+            "Description": "Test unpackaged scenarios for PushNotifications",
+            "Filename": "PushNotificationTests.dll",
+            "Parameters": "/name:UnpackagedTests#metadataSet1::VerifyForegroundHandlerFails",
+            "Architectures": ["x64"],
```

Comment:
Should this be enabled for arm64 at some point too?

Reply:
We can enable this for all archs but this is just the template for us. They are disabled due to WinAppSDK issue: https://github.com/microsoft/WindowsAppSDK/issues/3137#issuecomment-1332851404. I'll come back here with a better testdef in the future, but for now I am just laying down the basic foundation for WinAppSDK testing as a whole. I'll deal with notifications soon after.